### DISCUSSION TITLE
id is NaN

### DISCUSSION THREAD

**Discussion by basepack** [pods-framework/pods#6837] [Q&A]
Hi,

After saving a Pod I am redirected to the just saved Pod. However I get a message that the Pod is not found. 
When I look in URL I see that the ID is NaN:
`wp-admin/admin.php?page=pods-manage-omschrijving&action=edit&do=create&id=NaN`

How can that be the case?

Edit: when looking in the console. I see a POST to: wp-admin/admin-ajax.php?pods_ajax=1&action=pods_admin

With this data:

```
action:pods_admin
method:process_form
do:create
_pods_nonce:0bdba8efa8
_pods_pod:omschrijving
_pods_id:
_pods_uri:2e83d596c9
_pods_form:naam,omschrijving
_pods_location:/wp-admin/admin.php?page=pods-manage-omschrijving&action=add
publish:Nieuwe Omschrijving opslaan 
pods_field_naam:Test
pods_field_omschrijving:<p>Test</p>
```

Response is empty.

When editing a Pod I get an alert with: "Error: Unable to process request, please try again."

**Comment by Shelob9** [comment]
If the save had been successful, NaN would have been replaced with the new item ID. It looks like the operation timed out. Please increase the amount of memory allocated to PHP and try again.

**Comment by sc0ttkclark** [comment]
We should add code to detect NaN and throw an error instead of redirecting.
